### PR TITLE
feat: add taskmanager in start script

### LINKED ISSUE #1040: Wrap TaskManager script to start and auto-restart with mon
Add some scripts like `mon` to start TaskManager.

### PR DESCRIPTION
**Pull request by tobegit3hub**
* Refactor to add scripts to setup standalone and cluster 
* Support start and stop taskmanager
* Support mon in MacOS
* Resolve https://github.com/4paradigm/OpenMLDB/issues/1040

### CODE REVIEW

**Comment by tobegit3hub** [comment]
Here is the command to setup standalone openmldb.

Screen Shot 2022-01-10 at 6 11 19 PM

Here is the command to setup cluster openmldb in local server.

Screen Shot 2022-01-10 at 6 13 24 PM

**Review by dl239 — APPROVED** [review]

File: release/bin/boot_taskmanager.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ulimit -c unlimited
+ulimit -n 655360
+
+cd "$(dirname "$0")/../" || exit
+
+if [ -f "./conf/taskmanager.properties" ]; then
+  cp ./conf/taskmanager.properties ./taskmanager/conf/taskmanager.properties
+fi
+
+pushd ./taskmanager/bin/ > /dev/null || exit
```
Comment (aceforeverd) [inline release/bin/boot_taskmanager.sh:26]: use bash as shebang, or replace `pushd`, `popd`

File: release/bin/start.sh [hunk 2]
```diff
@@ -15,7 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export COMPONENTS="tablet nameserver apiserver"
+set -e
+
+if [[ $OSTYPE == 'darwin'* ]]; then
```
Comment (aceforeverd) [inline release/bin/start.sh:19]: so we should replace all shebang in script with `/bin/bash`, if there is no requirements that a env without bash installed.

Comment (tobegit3hub) [reply release/bin/start.sh:19]: I have tested /bin/sh supports `OSTYPE` as well.

**Review by aceforeverd — APPROVED** [review]

**Comment by tobegit3hub** [comment]
[Implementation of TaskManager](https://trello.com/c/EkxZMO7i/28-implementation-of-taskmanager)

**Pull request merged**